3.test_cases/pytorch/optimum-neuron/llama3/slurm/fine-tuning/train.py

```diff
@@ -0,0 +1,121 @@
+import os
+import torch
+import argparse
+from datasets import load_dataset
+from peft import LoraConfig
+from transformers import AutoModelForCausalLM, AutoTokenizer, set_seed
+from optimum.neuron import NeuronSFTConfig, NeuronSFTTrainer
+from optimum.neuron.distributed import lazy_load_for_parallelism
+import torch_xla.core.xla_model as xm
+
+HF_token = os.environ["HUGGINGFACE_TOKEN"]
```

Comment:
```suggestion
HF_token = os.environ["HF_TOKEN"]
```
Also there needs to be an explanation in readme if we need to export token as an environment variable.